ui-admin/src/study/families/FamilyActions.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Family } from '@juniper/ui-core'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import Api from 'api/api'
+import { Store } from 'react-notifications-component'
+import {
+  failureNotification,
+  successNotification
+} from 'util/notifications'
+import JustifyChangesModal from 'study/participants/JustifyChangesModal'
+import { useNavigate } from 'react-router-dom'
+
+/**
+ * Renders editable page with all family members and relations.
+ */
+export const FamilyActions = (
+  { family, studyEnvContext }: {
+    family: Family, studyEnvContext: StudyEnvContextT
+  }
+) => {
+  const [isDeleting, setIsDeleting] = React.useState<boolean>(false)
+  const navigate = useNavigate()
+
+  const deleteFamily = async (justification: string) => {
+    try {
+      await Api.deleteFamily(
+        studyEnvContext.portal.shortcode,
+        studyEnvContext.study.shortcode,
+        studyEnvContext.currentEnv.environmentName,
+        family.shortcode,
+        justification)
+      setIsDeleting(false)
+      Store.addNotification(successNotification('Family deleted.'))
+      navigate(`../../../participants?groupByFamily=true`)
```

Comment:
that's a few too many .. 😄   use the helper method instead
```suggestion
      navigate(`${participantListPath(...)}?groupByFamily=true` )
```